packages/ui/src/components/Forms/DateTimeInput/DateTimeInput.tsx

```diff
@@ -0,0 +1,48 @@
+import classNames from 'classnames'
+import { forwardRef, memo, Ref } from 'react'
+import { useComponentClassName } from '../../../auxiliary'
+import { toEnumStateClass, toEnumViewClass, toViewClass } from '../../../utils'
+import { TextInputOwnProps } from '../TextInput'
+import { FallbackDateTimeInput } from './FallbackDateTimeInput'
+
+export type DateTimeInputProps = TextInputOwnProps & {
+	type: 'date' | 'time' | 'datetime'
+	min?: string
+	max?: string
+}
+
+const isInputDateTimeLocalSupported = (() => {
```

Comment:
I am thinking it it wouldn't be better to have it as meoized value so it is not executed on load.

Reply:
This should run once when initialized.

Reply:
I can confirm it runs just once.

 In your example you have provided you used it inside of the component, however I already moved it outside of the render loop and this way it runs only once.